java-11/appdynamics/README.md

```diff
@@ -6,11 +6,43 @@ Basic Usage
 
 Se README for Java 11 for general usage.
 
+## Configuration of AppDynamics
+
+AppDynamics will automatically report itself by using the value of the
+environment variable `APP_NAME` as the Application name.
```

Comment:
`APPD` or `APP`?

Reply:
`APP_NAME` is provided today. It's a wee bit unfortunate if is removed in naiserator. Then it *can* simply be set in the Docker file though. 

But I think it is a very good practice to separate build from instance-name, and is something that should be injected at deploy time, and not at build time.